src/types.js

```diff
@@ -26,5 +26,25 @@ export const TYPES = {
   reviewGuidelines: {
     name: 'Review Guidelines',
     fileName: 'review_guidelines'
+  },
+  cookiesPolicy: {
+    name: 'Cookies Policy',
+    fileName: 'cookies_policy'
+  },
+  devPaymentsTos: {
+    name: 'Developer Payments Terms of Service',
+    fileName: 'dev_payments_tos'
+  },
+  communityPaymentsTos: {
+    name: 'Community Payments Terms of Service',
+    fileName: 'community_payments_tos'
+  },
+  commercialTerms: {
```

Comment:
> This type of document applies to _commercial entities_ (as opposed to _individuals_) using Facebook products.
> 
> Based on our hypothetical characterization of contractual document types as _provider_ / _user_  / _object_, it makes sense to create a new document type, because the _user_  changes.